src/news-feed.js

```diff
@@ -69,7 +68,7 @@ export default class NewsFeed {
                 }
 
                 // and ensure all the headlines are up to date
-                // Searlise this, so we only update one at a time
+                // Serialize this, so we only update one at a time
```

Comment:
Converting from English to American eh :-)

Reply:
unintentionally